ProjectUpdater/nbproject/project.properties

```diff
@@ -0,0 +1,108 @@
+annotation.processing.enabled=true
+annotation.processing.enabled.in.editor=false
+annotation.processing.processors.list=
+annotation.processing.run.all.processors=true
+annotation.processing.source.output=${build.generated.sources.dir}/ap-source-output
+application.title=ProjectUpdater
+application.vendor=sirius
+auxiliary.org-netbeans-spi-editor-hints-projects.perProjectHintSettingsFile=nbproject/cfg_hints.xml
+build.classes.dir=${build.dir}/classes
+build.classes.excludes=**/*.java,**/*.form
+# This directory is removed when the project is cleaned:
+build.dir=build
+build.generated.dir=${build.dir}/generated
+build.generated.sources.dir=${build.dir}/generated-sources
+# Only compile against the classpath explicitly listed here:
+build.sysclasspath=ignore
+build.test.classes.dir=${build.dir}/test/classes
+build.test.results.dir=${build.dir}/test/results
+# Uncomment to specify the preferred debugger connection transport:
+#debug.transport=dt_socket
+debug.classpath=\
+    ${run.classpath}
+debug.modulepath=\
+    ${run.modulepath}
+debug.test.classpath=\
+    ${run.test.classpath}
+debug.test.modulepath=\
+    ${run.test.modulepath}
+# This directory is removed when the project is cleaned:
+dist.dir=dist
+dist.jar=${dist.dir}/ProjectUpdater.jar
+dist.javadoc.dir=${dist.dir}/javadoc
+endorsed.classpath=
+excludes=
+includes=**
+jar.archive.disabled=${jnlp.enabled}
+jar.compress=false
+jar.index=${jnlp.enabled}
+javac.classpath=\
+    ${ant.core.lib}
+javac.external.vm=false
+javac.modulepath=
+javac.processormodulepath=
+# Space-separated list of extra javac options
+javac.source=11
+javac.target=11
```

Comment:
Is this line needed in this instance? If not, can probably be removed to save future pain when we update java

Reply:
Thanks @antares1470. I took the PR back into draft as I wanted to make some more changes but I was obviously too slow. Great feedback. I've fixed this so it does not show the target.

Reply:
Hmm it turns out this is a problem for the stand alone java projects. It was failing to compile without the target so I had to bring it back.